src/routes/profile-route/profile-fests-route.jsx

```diff
@@ -75,91 +119,87 @@ export const ProfileFestsRoute = () => {
           </div>
           <div className="flex flex-col">
             {!loading &&
-              data.fests.map((items, index) => {
-                const startDate = new Intl.DateTimeFormat("en-US").format(
-                  new Date(items.startDate)
+              data.fests.map((fest) => {
+                const startDate = new Intl.DateTimeFormat("en-GB").format(
+                  new Date(fest.startDate.replace(/-/g, "/"))
+                );
+                const endDate = new Intl.DateTimeFormat("en-GB").format(
+                  new Date(fest.endDate.replace(/-/g, "/"))
                 );
-                const endDate = new Intl.DateTimeFormat("en-US").format(
-                  new Date(items.endDate)
+                const attendeeStatus = fest.attendees.some(
+                  (attendee) => attendee.username === currentUser.username
                 );
-                // const isClicked = items.some((click) => click.id === items.id);
+                const festDateInFuture = dayjs().isBefore(fest.endDate);
+                // const isClicked = fest.some((click) => click.id === fest.id);
                 return (
                   <div
-                    key={index}
+                    key={fest.id}
                     className="flex flex-row justify-between mb-5 border-b border-black"
                   >
                     <div className="mb-5">
                       <h3 className="font-arialBold mb-2.5">
-                        <Link to={`/fests/${items.id}`}>{items.name}</Link>
+                        <Link to={`/fests/${fest.id}`}>{fest.name}</Link>
```

Comment:
Getting errors that Link is undefined

Reply:
Thanks! I'll comment on this one (it was like that before I started on this but still deserves a comment. Might be something the team was thinking to do in the future)

Reply:
This is set for a future task in Jira - I believe Sam is working on it. So when Josh merges everything this will be resolved.